### ISSUE TITLE
Typo in io.py causing  failure of 3D segmentation on tiff file with ".tiff" extension.

### ISSUE THREAD

**Issue by simonlsp** [MouseLand/cellpose#478]
Behavior:
Tiff file with extension ".tiff" is mistakenly detected as 2D images, they were read by cv2, and only axis 2, 1, 0 are used, making 3D segmentation impossible. Tiff files with extension ".tif" is ok.

Cause:
Typo in io.py as below.

File: io.py
function: imread
line: 75
original:
```python
if ext== '.tif' or ext=='tiff':
```
fixed:
```python
if ext== '.tif' or ext=='.tiff':
```

File: io.py
function: imread
line: 90
original:
```python
if ext== '.tif' or ext=='tiff':
```
fixed:
```python
if ext== '.tif' or ext=='.tiff':
```

Should I start a pull request, or you will fix it yourself?

**Comment by simonlsp** [comment]
A fix is submitted as Pull request #479.

**Comment by carsen-stringer** [comment]
thank you I fixed it! thank you though for the pull request, much appreciated!
